x-pack/filebeat/input/netflow/decoder/template/template.go

```diff
@@ -73,8 +75,15 @@ func (t *Template) Apply(data *bytes.Buffer, n int) ([]record.Record, error) {
 	if t.Length == 0 {
 		return nil, ErrEmptyTemplate
 	}
-	if n == 0 {
-		n = data.Len() / t.Length
+	maxRecords := 0
+	if t.Length > 0 {
```

Comment:
This `if` is redundant – we check for `t.Length == 0` above. This would also make the zero initialization of `maxRecords` unnecessary.

Reply:
i agree. i'll adjust this